doc/source/reference/random/change-log.rst

```diff
@@ -0,0 +1,39 @@
+Change Log for the original bashtage/randomgen repo
+---------------------------------------------------
+v1.16.1
+=======
+- Synchronized with upstream changes.
+- Fixed a bug in gamma generation if the shape parameters is 0.0.
+
+v1.16.0
+=======
+- Fixed a bug that affected ``randomgen.dsfmt.DSFMT`` when calling
+  ``~randomgen.dsfmt.DSFMT.jump`` or ``randomgen.dsfmt.DSFMT.seed``
+  that failed to reset the buffer.  This resulted in upto 381 values from the
+  previous state being used before the buffer was refilled at the new state.
+- Fixed bugs in ``randomgen.xoshiro512starstar.Xoshiro512StarStar``
+  and ``randomgen.xorshift1024.Xorshift1024`` where the fallback
+  entropy initialization used too few bytes. This bug is unlikely to be
+  encountered since this path is only encountered if the system random
+  number generator fails.
+- Synchronized with upstream changes.
+
+v1.15.1
+=======
+- Added Xoshiro256** and Xoshiro512**, the preferred generators of this class.
```

Comment:
rst interprets `**` as formatting text in bold

Reply:
Are you sure this is true w.r.t trailing whitespace:

This renders correctly in https://bashtage.github.io/randomgen/change-log.html